package.json

```
@@ -2,10 +2,11 @@
   "name": "svelte-collapsible",
   "description": "A collection of high-level Svelte components designed for expanding and collapsing content.",
   "keywords": ["svelte", "component", "accordion", "collapse", "expand"],
-  "version": "0.2.1",
+  "version": "0.2.2",
   "svelte": "src/index.js",
   "module": "dist/index.mjs",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

actually, would this have to be
```suggestion
  "types": "src/index.d.ts",
```
?

---

You right, this will need to point to `src`. If you build the package `npm run build` you can see that types are not included